pyhmy/validator.py

```diff
@@ -0,0 +1,651 @@
+import json
+
+from decimal import (
+    Decimal,
+    InvalidOperation
+)
+
+from .account import (
+    get_balance,
+    is_valid_address
+)
+
+from .exceptions import (
+    InvalidValidatorError,
+    RPCError,
+    RequestsError,
+    RequestsTimeoutError
+)
+
+from .numbers import (
+    convert_atto_to_one,
+    convert_one_to_atto
+)
+
+from .staking import (
+    get_all_validator_addresses,
+    get_validator_information
+)
+
+
+_default_endpoint = 'http://localhost:9500'
+_default_timeout = 30
+
+_name_char_limit = 140
+_identity_char_limit = 140
+_website_char_limit = 140
+_security_contact_char_limit = 140
+_details_char_limit = 280
+_min_required_delegation = 10000
```

Comment:
It appears as tho these are used only in the `Validator` class. Define them as class attributes for `Validator`, and we could probably make them public.